### DISCUSSION TITLE
Supported Domains  (TLD)

### DISCUSSION THREAD

**Discussion by Metallica22** [homeassistant-apps/app-cloudflared#302] [Q&A] [answered]
Hello, 
I've been using Cloudflared since the end of 2022.  
A really great extension, thanks for that.  
Shortly after my setup, a new HA version came out.  
In this, the following changes from Cloudflared were also announced.  
LIMITATIONS

Unusable TLDs

Due to a limitation in the Cloudflare API, you can not use this integration with any of the following TLD's:

 .cf
 .ga
 .gq
 .ml
 .tk

I have a ".TK" domain extension, can I update Cloudflared or will the extension no longer work after the Update?

**Comment by brenner-tobias** [comment]
Thanks for your question. I assume you are referring to the [Cloudflare HA Integration ](https://www.home-assistant.io/integrations/cloudflare/). So this add-on is not using the Cloudflare API but using the Cloudflare Tunnel for a connection. Now if you are using the Cloudflare Intetration, you have to reach out to them regarding the limitations for the TLDs. My best guess would be that you should not be able to use it, but wheather you update or not should not make a difference, since even the "old" version is using the API and if Cloudflare did changes there, it affects all the versions. They only updated the Documentation accordingly.

If you are using our add-on, I am not aware that they changed anything, but I am not sure to be honest. I just got a new .ga Domain and am waiting for it to be changed to Cloudflare Nameservers so I can check.

**Comment by Metallica22** [comment, marked as answer]
Thank you for your prompt reply.  
I use your addon.  
I've tested it myself now that I know they're different things.  
So my ".TK" address still works after the update.  

I would like to support the great work, is there a way to donate?

**Reply by brenner-tobias** [reply]
great, thanks for the update. No need to donate for this add-on, I suggest to support HomeAssistant in general where the heavy lifting is done ;-)
